### DISCUSSION TITLE
Custom Roblox, Steam, Farming Sim Player mod$ please

### DISCUSSION THREAD

**Discussion by omac777** [fishfolk/jumpy#323] [Ideas]
Just dropping by to say my son and I enjoyed it.  
Fun to play, great looking, great sounds and great music.

Some customized NON-FREE content could be made available and would be a great way to support the ecosystem for sure.
FishFight Customization As A Service:
-Customized Fish Player Character Look. When stuff happens unique entertaining looks and effects distinguishing it from other characters
-Customized Fish Player CharacterSounds. When stuff happens unique entertaining sound effects distinguishing it from other characters
-Customized Fish Player Music.  When other players on the internet encounter your fish player, the background music is customized to your selected playlist and they can hear it and adjust the volume for it.

Could FishFight be integrated/sold into Roblox somehow as a mod?  We do buy Robux.  
Could FishFight be integrated/sold into Steam and Farming Simulator as a mod?  We do buy Steam.
All this could help support your efforts.

Cheers and we wish the FishFight team health happiness and prosperity in the New Year.

**Comment by erlend-sh** [comment]
We’ll be doing all of the above in some way or another.

As for the integration with Roblox, I’d love to hear from Roblox players/modders on that.
